schemas/generate_model_event.py

```diff
@@ -79,8 +84,12 @@ def findNodeWithNodeId(self,root, nodeId):
         return node
 
     def checkNodeType(self, node):
-        if (node.tag == self.nameSpace + "UAObjectType") or (node.tag == self.nameSpace + "UAVariable") or (
-            node.tag == self.nameSpace + "UAObject") or (node.tag == self.nameSpace + "UAMethod") or (node.tag == self.nameSpace + "UAVariableType"):
+        if (
+                (node.tag == self.nameSpace + "UAObjectType") or
```

Comment:
I am not sure what would be faster/better, a compare "list" like this or using something with self.nameSpace.endswith(("UAObjectType","UAVariable", "UAObject","UAMethod","UAVariableType")).
It feels not DRY.

Reply:
this is not part of lirbary so need to need to opitmize. This is just run by library developers